hydrabus_framework/core/engine.py

```diff
@@ -0,0 +1,122 @@
+import inspect
+import os
+import pkgutil
+import sys
+from hydrabus_framework.core.logger import Logger
+from hydrabus_framework.core.command.run import run_module
+from hydrabus_framework.core.command.show import show
+from hydrabus_framework.core.command.set_options import set_options
+from hydrabus_framework.core.command.use import use
+from hydrabus_framework.core.command.back import back
+from hydrabus_framework.core.command.quit import hbf_exit
+from hydrabus_framework.core.command.help import hbf_help
+from hydrabus_framework.modules.base import BaseModule, ABaseModule
+from importlib import import_module
+from prompt_toolkit.shortcuts import PromptSession
+from prompt_toolkit.styles import Style
+
+
+class HydraFramework:
+    """
+    Framework core engine
+    """
+    def __init__(self):
+        self.logger = Logger()
+        self.app_path = sys.path[0]
+        self.current_module = BaseModule()
```

Comment:
It is useless to have an empty based module. You should handle the case when no module are selected

Reply:
Done [#10c3681](https://github.com/hydrabus-framework/framework/commit/10c368189272dfb79bbaec95279da987552428a0)